src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt

```diff
@@ -1182,8 +1183,10 @@ class NodeServiceTests {
       val expectedTimestamp =
         OffsetDateTime.parse(
             authCompletedEvent.data.timestampOperation, DateTimeFormatter.ISO_OFFSET_DATE_TIME)
+          .atZoneSameInstant(ZoneId.of("Europe/Paris"))
```

Comment:
please parameterize those tests with expected timestamp as done [here](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/0173a8ac15f439a3a90012647da9deb560fa6e75/src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeServiceTests.kt#L226-L228). This take into account both the expected string output format (making it hardcoded into tests) and Italy timezone variation (you can re-use the same method source as done per this test)

Reply:
Improved tests in 662ad21